### DISCUSSION TITLE
Toggle notes to flashcards

### DISCUSSION THREAD

**Discussion by Dmc-74** [st3v3nmw/obsidian-spaced-repetition#1015] [Q&A]
Complete novice here at using this plug in.  Is there a way to turn toggle notes already created into flash cards without having to start from scratch?

**Comment by ronzulu** [comment]
Best if you elaborate on your question.

What do you have now? What are you hoping for?

Have you seen the information about flashcards at https://www.stephenmwangi.com/obsidian-spaced-repetition/flashcards/

There is a feature request at: https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/729. Is that what you are after?

**Reply by Dmc-74** [reply]
To summarize my notes, I normally write a topic sentence / question then write the answer as a toggle in Obsidian so it’s hidden. Is there an easy way just to convert the topic sentence to the front of the flash card and the toggle answer to the back of the flash card?  There’s a feature like this to convert from Notion note to Anki

**Comment by ronzulu** [comment]
I'm understanding more now. The term "toggle" isn't an obsidian term and for me had other meaning.

So are you wanting to import notes that you've written outside of obsidian into obsidian?

This plug-in doesn't have a feature that would assist in this specific situation.

If you are able to define a "regular expression" that matches the structure of your existing text, there might be a way of automating the import process. This would require using tools outside of this plug-in.

If you do succeed in finding a solution, perhaps post your findings in this topic.

**Comment by Dmc-74** [comment]
Hi,I’m so sorry but I’m such a novice at Obsidian and have absolutely no background in computer language that I don’t quite understand a lot of the language.I create the notes in Obsidian. Example,Topic/question    > answers (this answer is a toggle and the answer is hidden.  I reveal the answer when I click on the symbol “>”.Thank you so much for your assistance,DiemOn Jul 9, 2024, at 8:21 PM, Ronny Zulaikha ***@***.***> wrote:﻿
I'm understanding more now. The term "toggle" isn't an obsidian term and for me had other meaning.
So are you wanting to import notes that you've written outside of obsidian into obsidian?
This plug-in doesn't have a feature that would assist in this specific situation.
If you are able to define a "regular expression" that matches the structure of your existing text, there might be a way of automating the import process. This would require using tools outside of this plug-in.
If you do succeed in finding a solution, perhaps post your findings in this topic.
